config/install/field.field.node.article.field_byline.yml

```diff
@@ -0,0 +1,28 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_byline
+    - node.type.article
+  module:
+    - allowed_formats
+    - text
+third_party_settings:
+  allowed_formats:
+    allowed_formats:
+      - basic_html
+id: node.article.field_byline
+field_name: field_byline
+entity_type: node
+bundle: article
+label: Byline
+description: 'Use the "P" button in the WYSIWYG editor to get '
```

Comment:
This looks incomplete...

Reply:
I've clarified this text in another commit.